dockerfiles/aegis-init-container/Dockerfile

```diff
@@ -0,0 +1,38 @@
+#
```

Comment:
:man_facepalming: some of these `Dockerfile`s were being `.gitignore`d for a long time.